.pre-commit-config.yaml

```diff
@@ -7,7 +7,6 @@ repos:
     - repo: https://github.com/pre-commit/pre-commit-hooks
       rev: v3.4.0
       hooks:
-          - id: trailing-whitespace
```

Comment:
I don't know where this whitespace is but since it fails while doing the bump2version, I'm tossing it to make my life easier for now.